code/game/atoms.dm

```diff
@@ -920,9 +920,22 @@ GLOBAL_LIST_EMPTY(blood_splatter_icons)
 		add_blood_overlay()
 	return TRUE //we applied blood to the item
 
-/obj/item/clothing/gloves/add_blood(list/blood_dna, b_color)
+/*
+This proc makes src gloves bloody, if you touch something with them you will leave a blood trace
+
+blood_dna: list of blood DNAs stored in each atom in blood_DNA variable or in get_blood_dna_list() on carbons
+b_color: blood color, simple. If there will be null, the blood will be red, otherwise the color you pass
+amount: amount of "blood charges" you want to give to the gloves, that will be used to make items/walls bloody.
+	You can make something bloody this amount - 1 times.
+	If this variable will be null, amount will be set randomly from 2 to max_amount
+max_amount: if amount is not set, amount will be random from 2 to this value, default 4
```

Comment:
```suggestion
  * This proc makes src gloves bloody, if you touch something with them you will leave a blood trace

  * blood_dna: list of blood DNAs stored in each atom in blood_DNA variable or in get_blood_dna_list() on carbons
  * b_color: blood color, simple. If there will be null, the blood will be red, otherwise the color you pass
  * amount: amount of "blood charges" you want to give to the gloves, that will be used to make items/walls bloody.
  	You can make something bloody this amount - 1 times.
	If this variable will be null, amount will be set randomly from 2 to max_amount
  * max_amount: if amount is not set, amount will be random from 2 to this value, default 4
```
Minor style thing!